solutions/swb-reference/src/environment/sagemakerNotebook/sagemakerNotebookEnvironmentLifecycleService.test.ts

```diff
@@ -129,9 +128,8 @@ describe('SagemakerNotebookEnvironmentLifecycleService', () => {
     expect(response).toEqual({ ...environment, status: 'PENDING' });
   });
 
-  test('Launch should return mocked id when mounting datasets', async () => {
+  test.skip('Launch should return mocked id when mounting datasets', async () => {
```

Comment:
@nguyen102 I'm pretty sure this test (and possibly the one above it) isn't doing what we think it is, so I marked it as `skip` but I could also remove it completely. We are mocking `environment` which gets passed into `sm.launch`, is never modified, gets sent back in the response, and then is checked against its original value. Is that accurate or am I maybe missing something?

Reply:
Let's not get rid of test cases. I think this and the test above it are important. Can you fix this test to do what you think it needs to? Why does this not do what it should?

Reply:
Not a fan of skipped tests ... Can we add a story to the backlog to come back and polish it if it's not testing the right thing?

Reply:
Spoke with @SanketD92. My code changes actually make this test obsolete, which is why it didn't appear to be doing anything. Removed the test now that it isn't needed anymore.

Reply:
Looks like we're mocking the `getDatasetsToMount` helper method in this test case and the one above to return `s3Mounts` and `iamPolicyDocument` values, when only one test case (this one) should have returned them (while the other returned empty strings).